### PR TITLE
ADEN-12076 Disabling ads for logged users

### PR DESCRIPTION
**Pull request by dmnsobczak**

### CODE REVIEW

File: src/platforms/f2/modes/f2-ads-mode.service.ts [hunk 1]
```diff
@@ -48,7 +47,6 @@ export class F2AdsMode implements DiProcess {
 			)
 			.execute()
 			.then(() => {
-				utils.logger('DJ', 'Pipeline done');
```
Comment (nandy-andy) [inline src/platforms/f2/modes/f2-ads-mode.service.ts:51]: 💎

File: src/platforms/shared/modes/no-ads/no-ads.mode.ts [hunk 2]
```diff
@@ -1,15 +1,41 @@
...
+	communicationService,
+	DiProcess,
+	eventsRepository,
+	identityHub,
+	jwpSetup,
+	liveConnect,
+	liveRampPixel,
+	PartnerPipeline,
+	userIdentity,
+} from '@wikia/ad-engine';
```
Comment (nandy-andy) [inline src/platforms/shared/modes/no-ads/no-ads.mode.ts:13]: Poor no ads bundle is becoming bigger :( Are Nef and his team aware of this upcoming change?

File: src/platforms/shared/modes/no-ads/no-ads.mode.ts [hunk 3]
```diff
@@ -1,15 +1,41 @@
...
 		this.removeAdSlotsPlaceholders();
 		this.noAdsDetector.addReasons(window.ads.context.opts.noAdsReasons);
 		this.dispatchJWPlayerSetupAction();
+
+		this.pipeline
+			.add(
+				userIdentity,
+				liveRampPixel.setOptions({ dependencies: [userIdentity.initialized] }),
+				ats,
+				audigent,
```
Comment (nandy-andy) [inline src/platforms/shared/modes/no-ads/no-ads.mode.ts:31]: I'm not sure if we're supposed to track logged-in users with Audigent 🤔

File: src/platforms/ucp-mobile/setup/dynamic-slots/ucp-mobile-slots-definition-repository.ts [hunk 4]
```diff
@@ -50,20 +50,12 @@ export class UcpMobileSlotsDefinitionRepository {
...
 		const hasPortableInfobox = !!document.querySelector('.portable-infobox');
 
-		return (
-			isSearch ||
-			isHome ||
-			hasPortableInfobox ||
-			(hasPageHeader && !hasFeaturedVideo) ||
-			allAdsAllowed
-		);
+		return pageType !== 'special' || hasPortableInfobox || (hasPageHeader && !hasFeaturedVideo);
```
Comment (nandy-andy) [inline src/platforms/ucp-mobile/setup/dynamic-slots/ucp-mobile-slots-definition-repository.ts:58]: There are special pages where TLB is applicable for example Special:Search 🤔

Comment (nandy-andy) [reply src/platforms/ucp-mobile/setup/dynamic-slots/ucp-mobile-slots-definition-repository.ts:58]: Oh, but then we're returning a `'search'` in `pageType` most likely 👌

**Review by nandy-andy — COMMENTED** [review]
The changes in this PR look more like cosmetic changes and the real turning ads off for logged-in users is meant to happen in the UCP: https://github.com/Wikia/unified-platform/pull/11758 We may need to rethink it, though 🤔

File: src/platforms/shared/modes/no-ads/no-ads.mode.ts (hunk 3 above)
Comment (dmnsobczak) [reply src/platforms/shared/modes/no-ads/no-ads.mode.ts:31]: That's what I wanted to give Nef's team to review 🤔

**Comment by dmnsobczak** [comment]
@nandy-andy This PR is also made to remove potential breaking points after cleanup in UCP :)

File: src/platforms/shared/modes/no-ads/no-ads.mode.ts (hunk 3 above)
Comment (kwfandom) [reply src/platforms/shared/modes/no-ads/no-ads.mode.ts:31]: Nef said 'yes, we want to track logged-in users', so that change is ok.

**Review by nandy-andy — APPROVED** [review]

**Pull request merged**